src/validators/model_fields.rs

```diff
@@ -184,6 +186,10 @@ impl Validator for ModelFieldsValidator {
                         // extra logic either way
                         used_keys.insert(lookup_path.first_key());
                     }
+                    if let Some(msg) = &field.deprecation_msg {
+                        let deprecation_warning_type = py.import_bound("builtins")?.getattr("DeprecationWarning")?;
```

Comment:
Try this instead:

```suggestion
                        let deprecation_warning_type = pyo3::exceptions::PyDeprecationWarning::type_object_bound(py);
```

Reply:
Updated. This works. Thanks! There's another `UserWarning` imported in the same way somewhere in the code base. I can create another PR to change that.

Reply:
That would be a great follow-up, yes please 👍